src/plotter_disk.hpp

```diff
@@ -36,8 +36,18 @@
 #define _HAS_STD_BYTE 0
 #define NOMINMAX
 
+#ifdef __APPLE__
+// std::filesystem is not supported on Mojave
 #include "../lib/include/filesystem.hpp"
 namespace fs = ghc::filesystem;
+#else
+#if __has_include(<filesystem>)
```

Comment:
if you really want to produce nice errors, you can check for the existence of this feature by:

```
#ifdef __has_include
#if __has_include(<...>)
#endif
#endif
```

Reply:
added to chia_filesystem.hpp. it tries <filesystem> if __has_include is missing.